packages/components/src/components/drawer/drawer.jsx

```diff
@@ -15,54 +16,45 @@ const IconDrawer = ({ className }) => (
     </svg>
 );
 
-class Drawer extends React.PureComponent {
-    constructor(props) {
-        super(props);
-        this.state = { is_open: props.is_open };
-    }
+const Drawer = ({
+    anchor = 'left',
+    children,
+    className,
+    is_open,
+    toggleDrawer,
+    contentClassName,
+    footer,
+    header,
+    width = 250,
+    zIndex = 4,
+}) => {
+    const [will_open, set_is_open] = useStateCallback(is_open);
```

Comment:
```suggestion
    const [is_open, setIsOpen] = useStateCallback(is_open);
```
As per our code convention, the setter function needs to be `camelCase`. We usually use `is` or `has` for boolean states

Reply:
As I see you changed name in state from `will_open ` to like it was `is_open ` . Here is a problem that - Identifier 'is_open' has already been declared . (23:4) That why I changed name in state